Fix python binary name

josephbisch opened the pull request:
Distros such as Arch that have Python 3 as the default Python version might have /usr/bin/python linked to python3 instead of python2.

openscap-jenkins commented:
Can one of the admins verify this patch?

josephbisch commented:
Actually this is incomplete. I notice that testcheck.py also just has `#!/usr/bin/python` as its shebang line, but is Python 2 code.

isimluk commented:
Thanks Jeseph, from me it is ack (to enable short-term build).

However in the long run, we should amend the scripts so these work under python2 or python3. We have done this in past for various tools (openscap-daemon, scap-as-rpm, oscap-docker, ...). In case, any of the scripts works in python3, please keep it /usr/bin/python.

Also, please rebase your patch on the latest master. There is a slight change of the location of some of the scripts.

josephbisch commented:
I rebased on master and I changed back to `#!/usr/bin/python` if the script doesn't use print, because the only issue seems to be the use of print as a statement instead of a function.
